src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,24 @@
+"""Kubectl Client BCI container"""
+
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package.versions import get_pkg_version
+
+KUBECTL_CONTAINERS = []
+KUBECTL_SUPPORTED_OS_VERSIONS = [
+    OsVersion.TUMBLEWEED,
+]
+
+for os_version in KUBECTL_SUPPORTED_OS_VERSIONS:
+    KUBECTL_CONTAINERS.append(
+        ApplicationStackContainer(
+            name="kubectl",
+            os_version=os_version,
+            version=get_pkg_version("kubernetes", os_version),
```

Comment:
please make this a placeholder so that not every version update is triggering a git commit.